packages/veritone-react-common/src/components/LogoDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import styles from './styles.scss';
+import Icon from 'material-ui/Icon';
+import IconButton from 'material-ui/IconButton';
+import ZoomOutMap from 'material-ui-icons/ZoomOutMap';
+import Select from 'material-ui/Select';
+import { MenuItem } from 'material-ui/Menu';
+import EngineOutputHeader from '../EngineOutputHeader';
+import PillButton from '../Parts/Buttons/PillButton';
+
+export default class LogoDetectionEngineOutput extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isExpanded: false
+        };
+
+        this.onItemSelected = this.onItemSelected.bind(this);
+        this.onEngineChanged = this.onEngineChanged.bind(this);
+        this.onExpandViewClicked = this.onExpandViewClicked.bind(this);
+    }
+
+    static propTypes = {
+        label: PropTypes.string,
+        data: PropTypes.array,
+        headerClassName: PropTypes.string,
+        bodyClassName: PropTypes.string,
+        itemClassName: PropTypes.string,
+        itemLabelClassName: PropTypes.string,
+        itemInfoClassName: PropTypes.string,
+        onItemSelected: PropTypes.func,
+        onEngineSelected: PropTypes.func,
+        onExpandViewClicked: PropTypes.func
+    };
```

Comment:
We need to also be able to pass in something like mediaPlayerTime or mediaPlayerPosition that indicates where the video is at so we can highlight the logo pills where media player time falls between start and end time. see https://veritone.invisionapp.com/d/main#/console/13611461/286589654/preview